src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/partials/PaymentBuilderPayoutsTotal/PaymentBuilderPayoutsTotal.tsx

```diff
@@ -0,0 +1,108 @@
+import { CaretDown } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import { BigNumber } from 'ethers';
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { useMemo, type FC } from 'react';
+
+import { accordionAnimation } from '~constants/accordionAnimation.ts';
+import useToggle from '~hooks/useToggle/index.ts';
+import Numeral from '~shared/Numeral/index.ts';
+import TokenIcon from '~shared/TokenIcon/index.ts';
+
+import {
+  type PaymentBuilderPayoutItem,
+  type PaymentBuilderPayoutsTotalProps,
+} from './types.ts';
+
+const PaymentBuilderTokensTotal: FC<PaymentBuilderPayoutsTotalProps> = ({
+  payouts,
+}) => {
+  const [isExpanded, { toggle }] = useToggle();
+  const sortedPayouts =
+    useMemo(
+      () =>
+        payouts?.sort((a, b) => {
+          if (
+            !a.amount ||
+            a.amount === '' ||
+            !b.amount ||
+            b.amount === '' ||
+            BigNumber.from(a.amount || '0').eq(BigNumber.from(b.amount || '0'))
```

Comment:
If `!a.amount` is false, `a.amount === ''` will also always be false, so we only need one

Reply:
Thanks for noticing, removed